.travis.yml

```diff
@@ -0,0 +1,14 @@
+branches:
+  except:
+    - master
+language: ruby
+cache: bundler
+sudo: false
+rvm:
```

Comment:
Really? I expected they would be using a newer Ruby by now.
Regardless, Guard in the bundle requires >= 2.2.3 and since travis installs the whole gem bundle, I can't test 2.1.x. I will limit it to just 2.2.5.

4328e2192592966f6e3a692e192431207df046fc